### ISSUE TITLE
Infinite scroll issue

### ISSUE THREAD

**Issue by TDMcKissick** [corygibbons/yuki#15]
Hey, I absolutely love this theme! But for the life of me I can not get infinite scroll to work on my blog. I have the "Click to load more posts" like on the preview theme, only difference is nothing loads when it's clicked. You can find my site at Escapekit.ca 

I appreciate any insight and help you may offer. 

Thanks!

**Comment by corygibbons** [comment]
@TDMcKissick Thanks!

Can you try disabling your share links addon to see if that fixes it? I have a feeling it may be causing problems because everything is working as expected when I've tested.

Also what browser are you using?

**Comment by TDMcKissick** [comment]
Hey Cory, 

I'm running on Chrome but I've also tested it on Firefox/Safari, and few other computers. I've tried removing the share it feature but still no luck. I went back through your versions and the one that fixes the scroll for custom url's works for the scroll but the photo sets show all and the copy vs just the one photo and the layout gets thrown off. 

Again I appreciate your help with this!
